network-api/networkapi/wagtailpages/customblocks.py

```diff
@@ -131,6 +143,15 @@ class Meta:
         template = 'wagtailpages/blocks/figure_grid_block.html'
 
```

Comment:
Can you add `group = 'Deprecated'` here so it shows up in the streamfield grouped as such? And for the figureBlock if that's the case?